about.html

```diff
@@ -0,0 +1,133 @@
+        <div>
+          <ul class="list-container">
+            <li class="list-item"><a href="./index.html">Home</a></li>
+            <li class="list-item"><a href="">See Product</a></li>
+            <li class="list-item"><a href="">Signup As A Talent</a></li>
+            <li class="list-item"><a href="">Sponsor</a></li>
+            <li class="list-item"><a href=""></a></li>
+          </ul>
+        </div>
+      </nav>
+      <section class="section-about">
+        <header class="header-about">
+          <p>Service Camp!</p>
+          <h1
+            >BRINGING SKILLS AND PRODUCTS TO YOUR DOORSTEP, NO MORE
+            MIDDLEMEN</h1
+          >
+        </header>
+        <article class="article-1">
+          <p
+            >No middle man, after searching for solutions we have built a
+            searchable directory of products and brands that are truly direct to
+            customers. We offer customers sustainable, valuable products through
+            low prices, high product quality, and outstanding customer
+            service.</p
+          >
+        </article>
+        <p class="reach"
+          >Please Reach out to us at: <span>servicecamp001@gmail.com</span>
+          <br />
+          If you would like to suggest a consumer brand for our directory</p
+        >
+      </section>
+      <section class="logo">
+        <header>
+          <h2>Service camp Goal and Logo</h2>
+          <span></span>
+        </header>
+        <p
+          >The idea is simple, avoid the costs of unnecessary middlemen-- all
+          the wholesalers, resellers and retilersnwho do little more than inflate
+          prices while clueless store clerks deliver a terrible experience to
+          customers. Instead, go straight to the source and connect directly with brands,
+          where you can enjoy a better experience, higher-quality products and
+          much lower prices.
+        </p>
+        <div>
+          <img src="./Images/service_camp.jpeg" alt="" />
+        </div>
+      </section>
+      <hr />
+      <section class="summit-history">
+        <header>
+          <h2>Service camp Logo and motto</h2>
+          <span></span>
+        </header>
+        <p>'We offer customers sustainable, valuable products through low prices, high product quality, and outstanding customerservice.'</p>
+        <div class="img">
+          <div class="first">
+            <h2>2013</h2>
+            <p>Lorem ipsum dolor sit amet.</p>
+          </div>
+          <div class="second">
+            <h2>2013</h2>
+            <p>Lorem ipsum dolor sit amet.</p>
+          </div>
+        </div>
+      </section>
+      <section class="partner">
+        <header>
+          <h2>Partner</h2>
+          <span></span>
+        </header>
+        <div class="partners-grid">
+          <p>mozilla</p>
+          <p>Google</p>
+          <p>NAVER</p>
+          <p class="row">daumkakao</p>
+          <p class="row-2">airbnb</p>
+        </div>
```

Comment:
- [ ] Please make sure to delete partners section in desctop screens and show it only in mobile size: 

Your design  | Behance design
------------- | -------------
<img width="822" alt="Screenshot 2022-04-15 at 21 13 46" src="https://user-images.githubusercontent.com/77038610/163600483-6fe58e29-bdae-49b9-adb8-34e575fa734c.png"> | <img width="810" alt="Screenshot 2022-04-15 at 21 14 09" src="https://user-images.githubusercontent.com/77038610/163600516-dd13a4a3-5fc4-485b-8818-79072c0ec155.png">

- [ ] Please make sure that the footer's background in about page, is dark in desktop size and light in mobile size  as provided in design.

Reply:
done